src/SprykerEco/Zed/Afterpay/Business/Payment/Transaction/Authorize/PaymentAuthorizeReaderInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Afterpay\Business\Payment\Transaction\Authorize;
+
+interface PaymentAuthorizeReaderInterface
```

Comment:
How useful is the `interface` without **any** available methods?  Or this is a `WIP` placeholder and *something* will be added later?